src/businesschat.cs

```diff
@@ -0,0 +1,44 @@
+//
+// BusinessChat bindings
+//
+// Authors:
+//     Manuel de la Peña <mandel@microsoft.com>
+//
+// Copyright 2017 Microsoft Inc. All rights reserved.
+//
+using System;
+using System.ComponentModel;
+
+using XamCore.Foundation;
+using XamCore.ObjCRuntime;
+#if MONOMAC
+using XamCore.AppKit;
+#else
+using XamCore.UIKit;
+#endif
+
+namespace XamCore.BusinessChat {
+	[Mac (10,13, onlyOn64: true), iOS (11,3)]
+#if MONOMAC
```

Comment:
You can reduce the number of `#if MONOMAC` here since you are just using one on the top just use

```
#if MONOMAC
using XamCore.AppKit;
using UIControl = XamCore.AppKit.NSControl;
#else
using XamCore.UIKit;
#endif
```

and just have `[BaseType (typeof (UIControl))]` for  `BCChatButton`.